### ISSUE TITLE
Data not present in pydot graph. 

### ISSUE THREAD

**Issue by lakhanjadhav1** [ChrisCummins/ProGraML#207]
## To Reproduce

1. Generate programl graph from any code
2. Convert that graph to dot using programl.to_dot()
3. That generated dot string doesn't have features info

## Environment

- ProGraML version (if building from source, use `git rev-parse HEAD`):
- How you installed ProGraML (source, pip):
- OS:
- Python version:
- Build command you used (if building from source):
- GCC/clang version (if building from source):
- Bazel version (if building from source):
- Versions of any other relevant libraries:

**Comment by ChrisCummins** [comment]
You're right, `pg.to_dot(G)` only adds the text of the node when exporting to dotfile.

Is there a specific feature you're looking to serialize?

You would need to modify this function and build from source to embed extra features:

https://github.com/ChrisCummins/ProGraML/blob/cd7d2939d485d989a9b105723fed989bcaca1f94/programl/graph/format/graphviz_converter.cc#L170-L201

I could see that this would be generally useful. Patches welcome!

Cheers,
Chris
